### PR TITLE
Add CLI config command to manage configs

### PR DESCRIPTION
**Pull request by cliveseldon**
- Adds command to manage configs
- Adds add, remove, list, activate, deactivate subcommands

Example:

```
$ seldon config list
config		path						active
------		----						------
kind-sasl	/home/work/seldon/cli/config-sasl.json	*

$ seldon config deactivate kind-sasl

$ seldon config list
config		path						active
------		----						------
kind-sasl	/home/work/seldon/cli/config-sasl.json	

$ seldon config add gcp-scv2 ~/work/seldon/cli/gcp.json

$ seldon config list
config		path						active
------		----						------
gcp-scv2	/home/work/seldon/cli/gcp.json		
kind-sasl	/home/work/seldon/cli/config-sasl.json	

$ seldon config activate gcp-scv2

$ seldon config list
config		path						active
------		----						------
gcp-scv2	/home/work/seldon/cli/gcp.json		*
kind-sasl	/home/work/seldon/cli/config-sasl.json

$ seldon config list kind-sasl
{
  "controlplane": {
    "schedulerHost": "172.19.255.2:9004"
  },
  "kafka": {
    "bootstrap": "172.19.255.3:9093",
    "caPath": "/home/work/gcp/scv2/certs/seldon-cluster-ca-cert/ca.crt"
  }
}
```

### CODE REVIEW

File: operator/pkg/cli/configs.go [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+	}
+}
+
+func (sc *SeldonCLIConfigs) Add(key string, configPath string) error {
+	sc.Configs[key] = configPath
+	return sc.save()
+}
+
+func (sc *SeldonCLIConfigs) Remove(key string) error {
+	delete(sc.Configs, key)
```
Comment (sakoush) [inline operator/pkg/cli/configs.go:120]: do we need to deactivate in the case it is active before removing?

File: operator/pkg/cli/configs.go [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+func (sc *SeldonCLIConfigs) List(key string) error {
+	if key == "" {
+		return sc.listConfigs()
+	} else {
+		return sc.listConfig(key)
+	}
+}
+
+func (sc *SeldonCLIConfigs) Add(key string, configPath string) error {
+	sc.Configs[key] = configPath
```
Comment (sakoush) [inline operator/pkg/cli/configs.go:115]: do we need to check if it already exists? do we want to overwrite?

File: operator/pkg/cli/configs.go [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+	return sc.save()
+}
+
+func (sc *SeldonCLIConfigs) Remove(key string) error {
+	delete(sc.Configs, key)
+	return sc.save()
+}
```
[the remark this answers is not in the document]
Comment (cliveseldon) [reply operator/pkg/cli/configs.go:122]: not sure we need a guard?

File: operator/pkg/cli/configs.go [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+			if err != nil {
+				return err
+			}
+		} else {
+			_, err = fmt.Fprintf(writer, "%s\t%s\t%s\n", k, v, "")
+			if err != nil {
+				return err
+			}
+		}
+	}
```
[the remark this answers is not in the document]
Comment (cliveseldon) [reply operator/pkg/cli/configs.go:85]: well that is why I am using a tabwriter - not sure above that

File: operator/pkg/cli/configs.go (hunk 2 above)
Comment (cliveseldon) [reply operator/pkg/cli/configs.go:115]: I was thinking not to keep it simple at present

File: operator/pkg/cli/configs.go (hunk 1 above)
Comment (cliveseldon) [reply operator/pkg/cli/configs.go:120]: good catch - will update

File: docs/source/contents/cli/index.md [hunk 5]
```diff
@@ -93,6 +93,51 @@ An example below shows an example where we connect via TLS to the Seldon schedul
...
+
+$ seldon config list kind-sasl
+{
+  "controlplane": {
+    "schedulerHost": "172.19.255.2:9004"
+  },
+  "kafka": {
+    "bootstrap": "172.19.255.3:9093",
+    "caPath": "/home/work/gcp/scv2/certs/seldon-cluster-ca-cert/ca.crt"
+  }
```
[the remark this answers is not in the document]
Comment (cliveseldon) [reply docs/source/contents/cli/index.md:137]: the above are not envars. If you mean the possible values that's already on the page regards keys.

**Review by sakoush — APPROVED** [review]

**Pull request merged**